script/Ciel/Day 6/06_01_CIEL06_7.txt

```diff
@@ -302,9 +302,10 @@ Despite being on the receiving end of his murderous gaze,
 -- Page 273, Offset 13875.
 -- 「まあいい、終わった事をウダウダ言っても時間の無駄だ。人生残り少ねぇんだし前向きにいかねえとな。
 // This guy _is_ Laurentis, right? 17 > 12 is I assume the problem here, and so that 人生残り is his own
+// spilled or spilt?
```

Comment:
spilt in america but i wouldn't look twice at spilled

Reply:
spilt it is then